impeller/entity/contents/text_contents.cc

```diff
@@ -4,6 +4,7 @@
 
 #include "impeller/entity/contents/text_contents.h"
 
+#include <iostream>
```

Comment:
```suggestion
```

Not sure if github is showing that right, but what I'm trying to articulate is that this line can be deleted.

Reply:
I got it!